tests/scripts/minikube.sh

```diff
@@ -74,6 +70,7 @@ case "${1:-}" in
     minikube_ssh
     ;;
   update)
+    source "${scriptdir}/../../build/common.sh"
```

Comment:
oh, you only get private images now if you run `update`?

Reply:
99% of the time i start minikube I want to use my local build so it has been nice that `up` will copy the images. Why can't `up` continue to copy the images?